index.html

```diff
@@ -0,0 +1,234 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>AirBnB</title>
+  <link rel="stylesheet" href="./styles/main.css">
+  <link rel="shortcut icon" href="images/logo__small.svg" type="image/svg">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header">
+    <section class="header__container">
+      <a href="#">
+        <img
+          src="images/logo__small.svg"
+          alt="logo"
+          class="header__container--logo"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a href="#" class="nav__list-link">
+              Become a host
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__list-link">
+              Trips
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__list-link">
+              Messages
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="" class="nav__list-link">
```

Comment:
```suggestion
            <a href="#" class="nav__list-link">
```
don't leave empty attributes